tools/clang/test/HLSLFileCheck/hlsl/intrinsics/wave/repacking.hlsl

```diff
@@ -0,0 +1,241 @@
+// RUN: %dxc -T lib_6_5 %s | FileCheck %s
+
+// Check that results of wave intrinsics are not re-used
+// cross DXR repacking points such as TraceRay() or CallShader();
```

Comment:
This mentions TraceRay, but never tests it. Are there other repacking points?

Reply:
I originally used CallShader because it requires the least boilerplate, but I agree we should test them all.
I've added TraceRay tests, and will later take a look at ReportHit.

Reply:
I'm now testing all the repacking point kinds: TraceRay, CallShader, ReportHit.